playwright/__init__.py

```diff
@@ -13,15 +13,17 @@
 # limitations under the License.
 
 from playwright.playwright import playwright
+import playwright.helper as helper
```

Comment:
lets export it in the root import, so that the users don't have to use the files directly?